src/server/utils/wallets/createLocalWallet.ts

```diff
@@ -13,7 +13,7 @@ interface CreateLocalWallet {
  * Create a local wallet with a random private key
  * Does not store the wallet in the database
  */
-export const createLocalWallet = async () => {
+export const createRandomLocalWallet = async () => {
```

Comment:
Nit: "random" sounds unclear to me (especially since the other analogous KMS ones don't have it). createLocalWallet was fine, or generateLocalWallet?